doc/MANIFEST.md

```diff
@@ -242,12 +326,16 @@ The static list of flags for your command
 
 It declares a `--human` flags with a short form: `-H`
 
+#### checkFlags
```

Comment:
I feel like the documentation assumes that the user can make a clear distinction between arguments and flags of a command line. It's a bit blurry to me. It could be cool to explain it here.

Reply:
Added paragraph to explain the argument and flag concept